src/urllib3/request.py

```diff
@@ -1,6 +1,9 @@
 from __future__ import absolute_import
 
+import sys
+
```

Comment:
Can you please remove those two lines too? Sorry that I did not use inline comments before.

Reply:
We need to `import sys` because we're calling `sys.modules` . Want me to change the style to `from sys import modules` to be more consistent with other imports throughout urllib3 instead?

Reply:
I misread and read six instead 🤦